lib/kennel/syncer.rb

```diff
@@ -146,6 +146,7 @@ def delete_matching_expected(a)
       end
 
       e = @lookup_map["#{a.fetch(:api_resource)}:#{a.fetch(:id)}"] || @lookup_map[tracking_id(a)]
+      return if e && a[:api_resource] != e.class.api_resource
```

Comment:
comment on what this does

Reply:
so this is the magic that makes it not match the incorrect class type allowing both old Dash and new Dashboard to work. otherwise because the tracking id is the same it would match the wrong object model and fail.